### PR TITLE
feat: added new first inquiry echo flag

### PR DESCRIPTION
**Pull request by lilyfromseattle**
[PURCHASE-2240]

This PR adds an echo flag for the new first inquiry flow

[PURCHASE-2240]: https://artsyproduct.atlassian.net/browse/PURCHASE-2240

### CODE REVIEW

File: Echo.json5 [hunk 1]
```diff
@@ -20,6 +20,7 @@
     { name: 'AROptionsLotConditionReport', value: true },
     { name: 'AROptionsEnableSales', value: true },
     { name: 'AREnableViewingRooms', value: true },
+    { name: 'AROptionsNewFirstInquiry', value: true },
```
Comment (erikdstock) [inline Echo.json5:23]: This is my first echo pr - What does `true` mean here? If that's what we want, ✅ .

**Review by erikdstock — APPROVED** [review]
Tentative approval if my review assumptions are correct.

File: Echo.json5 (hunk 1 above)
Comment (zephraph) [reply Echo.json5:23]: `true` in this context means that the feature is enabled.

**Review by zephraph — APPROVED** [review]
Looks great @lilyfromseattle, thanks for the quick turn around!

**Pull request merged**